dev/NavigationView/NavigationView.cpp

```diff
@@ -1512,6 +1512,7 @@ void NavigationView::UpdatePaneLayout()
                         {
                             const auto footersActualHeight = footerItemsRepeater.ActualHeight();
                             const auto menuItemsActualHeight = menuItems.ActualHeight();
+                            const auto separatorLineHeight = GetItemSeparatorHeight();
```

Comment:
I'm not totally sure this is the best way to get separator line height. @karenbtlai thoughts?:)